content/english/secret-messages/activity-2.md

```diff
@@ -5,6 +5,8 @@ weight: 3
 draft: false
 ---
 
+{{< youtube umCdJDmdf3s >}}
```

Comment:
Can you please use the following template instead to ensure the videos are of the correct size:

```suggestion
<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/umCdJDmdf3s" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
```